orchestration/src/main/java/com/sap/ai/sdk/orchestration/OrchestrationAiModelParameters.java

```diff
@@ -0,0 +1,86 @@
+package com.sap.ai.sdk.orchestration;
+
+import java.util.Map;
+import javax.annotation.Nonnull;
+
+/** Helper interface to define typed parameters. */
+@FunctionalInterface
+public interface OrchestrationAiModelParameters {
+  /**
+   * Get the parameters.
+   *
+   * @return the parameters.
+   */
+  @Nonnull
+  Map<String, Object> getParams();
+
+  /** GPT model parameters. */
+  interface GPT extends OrchestrationAiModelParameters {
+    /**
+     * Create a new builder.
+     *
+     * @return the builder.
+     */
+    @Nonnull
+    static GPT.Builder0 params() {
```

Comment:
Considering all parameters are optional, why is this a builder?

Reply:
> Considering all parameters are optional

I was under the assumption, all parameters are required.

Reply:
no, those are just a somewhat random selection to give examples